### ISSUE TITLE
Drop-down of saved scenarios in import/export section can be improved

### ISSUE THREAD

**Issue by mabijkerk** [quintel/etengine#1280]
When I go to the Flexibility -> [Import/Export](https://pro.energytransitionmodel.com/scenario/flexibility/electricity_import_export/interconnector-1) section, I can select price curves from saved scenarios.

<img width="468" alt="image" src="https://user-images.githubusercontent.com/67338510/196359947-ddb65dfa-3bc6-4e84-bb35-b670b82a3e8a.png">

When I try to select a saved scenario there are however two issues:

1. There does not seem to be a logical ordering of scenario titles
2. The list of saved scenarios is not exhaustive i.e., some of my saved scenarios are missing

<img width="488" alt="image" src="https://user-images.githubusercontent.com/67338510/196360238-0045af5f-dfe5-4641-9c20-c98ec3ca24e1.png">

@antw would you be able to have a look at this please, to see if the drop-down can be improved?

**Comment by antw** [comment]
Here's how it works today:

1. No saved scenarios for the current area code are shown. If your current scenario is `nl2019` no saved scenarios for `nl2019` will be included in the list.
2. At the top of the list are scenarios with the same end year as your current scenario, ordered with the most recently updated first.
3. Below those are the rest of your saved scenarios ordered by end year and then by most recently updated.

I think 2) and 3) are somewhat confusing.

@mabijkerk What do you think would be a more sensible ordering of scenarios? By title? Most recently updated first (regardless of end year)? Something else?

**Comment by AlexanderWirtz** [comment]
Thx for the explanation, @antw . 

I prefer option 1. Most of all, I would like a search function too, but I assume that is a bit more time consuming. 

Here's my current bind: I have a recent German scenario whose curve I want to use for a Dutch scenario. I know it exists in 'My scenarios', but cannot find it in the endless list of scenarios I get now. Even now that I know and understand the current sorting logic. 

I hope the new sorting method will fix this inability to find a scenario, but I am not sure. Hence the hope that we might implement a search function.

**Comment by mabijkerk** [comment]
Right, I forgot about areas. Taking areas into account my sorting preference would be: `area` > `end year` > `scenario title (alphabetically)`.
